backend/chainlit/auth.py

```diff
@@ -1,6 +1,6 @@
 import os
 from datetime import datetime, timedelta
-from typing import Any, Dict
+from typing import Any, Dict  # noqa: F401
```

Comment:
Referenced in the comment on line 52.

Reply:
Could you please specify the type with Python type annotations so that the noqa won't be necessary?

Reply:
Done ✅